### DISCUSSION TITLE
Move (100s of) QSOs to a different logbook?

### DISCUSSION THREAD

**Discussion by ei8fdb** [magicbug/Cloudlog#3233] [Q&A]
I've been logging my POTA, SOTA activities to separate logbooks (i.e.  POTA activations, SOTA activations), but now realise this isn't the best idea. It's common that landmarks; summits parks, beaches etc can be park of multiple programs. E.g. many summits are located in parks.

So now I'd like to move all my POTA, SOTA QSOs into 1 logbook. Ideally I'd like to do this in bulk.

There was a previous question about moving a QSO from a logbook/location (discussion https://github.com/magicbug/Cloudlog/discussions/3078), but the answer provided covered just moving location, _not_ logbook.

Does anyone have any suggestions on how to do this?

**Comment by alexjj** [comment]
I don't have an answer to your question (I'd wondered about something like phpmyadmin to edit the database), but I'm now considering doing the same thing. I used to do one Station Location per SOTA summit but it's a lot of hassle, and particularly when they overlap. 

How are you recording which summit or park you activated from in the QSO? In notes?

**Comment by magicbug** [comment]
I've not got a solution for bulk, you could just try doing it with SQL but risk something going wrong sadly.
